compute_funcx/endpoint/PyPI.md

```diff
@@ -0,0 +1,11 @@
+# Deprecation Warning
+
+This package is deprecated, and currently just wraps the [Globus Compute Endpoint](https://pypi.org/project/globus-compute-endpoint/) package with FuncX names. Users are recommended to upgrade to Globus Compute instead of using this package.
+
+# FuncX Endpoint
+
+[FuncX](https://globus-compute.readthedocs.io/en/latest/) is a distributed Function as a Service (FaaS) platform that enables flexible, scalable, and high performance remote function execution. Unlike centralized FaaS platforms, FuncX allows users to execute functions on heterogeneous remote computers, from laptops to campus clusters, clouds, and supercomputers.
+
+This package provides the functionality for running and managing [endpoints](https://globus-compute.readthedocs.io/en/latest/endpoints.html), which are the services that manage the execution of functions on target machines.
+
+If you want to submit functions to your own endpoints or to endpoints managed by someone else, you'll need the companion [FuncX SDK](https://pypi.org/project/funcx/) package.
```

Comment:
- Branding-wise, I believe the capitalization is `funcX` not `FuncX`.  Bit of a moot point now as we're, uh, rebranding.  "But still."  Heh.
- I submit that "Your own endpoints or ..." is superfluous.  Perhaps:
    > To submit functions for execution on compute endpoints, use the companion [funcX SDK](https://pypi.org/project/funcx/).